### ISSUE TITLE
How to train on single GPU ?

### ISSUE THREAD

**Issue by ThorPham** [Yunfan-Li/Twin-Contrastive-Learning#1]
Thank you for great job. I want to train on single GPU. Can i do that ?

**Comment by Yunfan-Li** [comment]
Maybe you could set --nproc_per_node=1.

**Comment by ThorPham** [comment]
@Yunfan-Li  thank you for your support . I have bug like this 
![Screenshot from 2022-07-29 22-50-22](https://user-images.githubusercontent.com/33828708/181797094-6c7b258d-308f-4388-840d-334e7d3abb24.png)

**Comment by ThorPham** [comment]
sorry my bad. I write 2 name python in cmd

**Comment by Breeze-J** [comment]
> sorry my bad. I write 2 name python in cmd
Could you please solve this problem?
I have another question, is it okay to just change 4 to 1?
OMP_NUM_THREADS=1 python -m torch.distributed.launch --nproc_per_node=1  train.py
Will it work if changed to the above?

**Comment by Yunfan-Li** [comment]
Yes. You could change --nproc_per_node according to the number of GPUs available.
